src/components/Pages/Student/StudentListener.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * {StudentListener} is an adapter between Geogebra and server.
+ * It dispatches/responds to events from the server related to manipulation
+ * of Geogebra elements and workshops. It uses API provided by [StudentClient].
+ */
+
+import api from '../../../feathers-client';
+
+// Geogebra does not support
+// superscript. Therefore, we use
+// Unicode to display them.
+const numberToSuper = {
```

Comment:
Sorry for being picky, but don't we have some utils package to put such things in? It's not actually scoped to "StudentListener" logically, at least it doesn't seem like it should be

Reply:
I assumed here the C-style structure a la Dennis and Richie. This stuff is not technically scoped to StudentListener but it is logically - it is the only class using this code and I actually moved it here from the class' method. If any need arises to re-use this code (I doubt it) I will move it elsewhere.